tests/unit/test_function.py

```diff
@@ -48,6 +48,52 @@ def test_integral_spline_interpolation(request, func, a, b):
     )
 
 
+@pytest.mark.parametrize(
+    "x,y,z_expected",
+    [
+        (0, 0, 1),
+        (1, 0, 0),
+        (0, 1, 0),
+        (0, 0, 1),
+        (0.5, 0.5, 1 / 3),
+        (0.25, 0.25, 25 / (25 + 2 * 5**0.5)),
+        ([0, 0.5], [0, 0.5], [1, 1 / 3]),
+    ],
+)
+def test_2d_shepard_interpolation(x, y, z_expected):
+    """Test the shepard interpolation method of the Function class."""
+    # Test plane x + y + z = 1
+    source = [(1, 0, 0), (0, 1, 0), (0, 0, 1)]
+    func = Function(source=source, inputs=["x", "y"], outputs=["z"])
```

Comment:
please consider to explicitly defining the interpolation method as the `shepard` method, this may no longer be the only, default option in the future.

Same thing happens in the second test function.